docs/api.rst

```diff
@@ -1040,6 +1040,7 @@ to handle it, which defaults to print a traceback and ignoring the exception.
     - nickname
     - roles
     - pending
+    - timeout
```

Comment:
Please note that this isn't called when a timeout expires.

I believe this should also include guild avatar but i am not fully sure.

Reply:
https://github.com/DisnakeDev/disnake/pull/510/commits/ec70df31904bdc8dcc362712cc8bfd149fec5974